gufe/components/solventcomponent.py

```diff
@@ -5,6 +5,7 @@
 from typing import Optional, Tuple
 
 from openff.units import unit
+from pint import Quantity
```

Comment:
Looks like this is the way (someone double check me), we'll need to switch the imports downstream when we do this.

Reply:
I don't think this is necessary at all. It looks like the line above for defining `unit` gives access to `unit.Quantity` for `openff-units` 0.3.0. Mixing imports from `pint` (even if this is literally all `unit.Quantity` is) is confusing at best to me.

What's the motivation?

Reply:
Recommend `from openff.units import Quantity` (or `from openff.toolkit import Quantity` where it's installed)

Reply:
@mattwthompson importing Quantity from `openff.units` is causing weird behaviour with mypy (see: https://github.com/OpenFreeEnergy/gufe/actions/runs/14599580786/job/40954126788?pr=533). Do you happen to know why that might be?